osu.Game/Tests/Visual/ScrollingTestContainer.cs

```diff
@@ -99,7 +99,7 @@ public double GetDisplayStartTime(double originTime, float offset, double timeRa
             public float GetLength(double startTime, double endTime, double timeRange, float scrollLength)
                 => implementation.GetLength(startTime, endTime, timeRange, scrollLength);
 
-            public float PositionAt(double time, double currentTime, double timeRange, float scrollLength)
+            public float PositionAt(double time, double currentTime, double timeRange, float scrollLength, double? originTime = null)
                 => implementation.PositionAt(time, currentTime, timeRange, scrollLength);
```

Comment:
Should fully proxy to the implementation:
```suggestion
            public float PositionAt(double time, double currentTime, double timeRange, float scrollLength, double? originTime = null)
                => implementation.PositionAt(time, currentTime, timeRange, scrollLength, originTime);
```